pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTAdditiveExpression.java

```diff
@@ -10,21 +10,37 @@
  * This has a precedence greater than {@link ASTShiftExpression}, and lower
  * than {@link ASTMultiplicativeExpression}.
  *
- * <p>Note that the children of this node are not necessarily {@link ASTMultiplicativeExpression},
- * rather, they are expressions with an operator precedence greater or equal to MultiplicativeExpression.
  *
- * <pre>
+ * <pre class="grammar">
```

Comment:
Is this css class `grammar` defined somewhere? Is it used by the javadoc view of your rule designer?

Reply:
Yes I added it for the designer's javadoc rewriter. It's rendered as an html header. 

We could achieve the same thing with a custom javadoc tag, but IntelliJ's inline javadoc viewer doesn't support them and doesn't render the content of the section, which is not fine for me.